Src/Newtonsoft.Json/JsonTextWriter.cs

```diff
@@ -785,9 +775,7 @@ public override void WriteComment(string? text)
         {
             InternalWriteComment();
 
-            _writer.Write("/*");
-            _writer.Write(text);
-            _writer.Write("*/");
```

Comment:
Such calls here are intentionally to avoid string allocations when doing string interpolation.

Reply:
Cancelled all such commits